samples/tool-guardrails/src/main/java/io/quarkiverse/langchain4j/sample/guardrails/EmailFormatValidator.java

```diff
@@ -56,7 +56,7 @@ public ToolInputGuardrailResult validate(ToolInputGuardrailRequest request) {
             return ToolInputGuardrailResult.success();
 
         } catch (Exception e) {
-            return ToolInputGuardrailResult.failure(
+            return ToolInputGuardrailResult.fatal(
```

Comment:
To answer the question, which is not asked yet: API of ToolInputGuardrailResult was changed[1] recently, and it looks like `fatal` failure is better describes the intention here. Otherwise, we will need to get rid of either exception or custom message.

[1] https://github.com/quarkiverse/quarkus-langchain4j/pull/1993